### DISCUSSION TITLE
Aubess 20A EU smart plug recognize as BSD29

### DISCUSSION THREAD

**Discussion by c0mplex1** [Koenkk/zigbee2mqtt#18205] [Devices]
Recently I bought 2pcs Aubess 20A EU Smart Plugs.
https://nl.aliexpress.com/item/1005003970217316.html
But when I pair them they are not recognized as such. They report themselves as Model BSD29 and not as TS011F_plug_1 as I would expect.
Can this be fixed?

Using Zigbee2mqtt version1.32.0-2

Simple name: 0x70b3d52b6002763f
Description
Last seen: N/A
Available: Disabled
Device Type: Router
Zigbee model: TS011F
Zigbee Manufacturer: _TZ3000_okaz9tjs
Description: Smart plug (with power monitoring by polling)
Support Status: Supported
IEEE address: 0x70b3d52b6002763f
Network address: 0x8A92
Firmware build date: 20210625
Firmware Version: 500+TZSKT31BS102
Manufacturer: TuYa
Model: BSD29

**Comment by permport** [comment]
Exactly the same issue here
Also not possible to perform an update
![image](https://github.com/Koenkk/zigbee2mqtt/assets/28897727/d5e2cd44-8d7c-4d98-a395-53eb957927eb)
![image](https://github.com/Koenkk/zigbee2mqtt/assets/28897727/5e2789fe-1792-4fad-b869-e324061f9900)

**Comment by joeShuff** [comment]
I'm seeing similar with the [UK version](https://www.aliexpress.com/item/1005005618210766.html). 
![image](https://github.com/Koenkk/zigbee2mqtt/assets/15524362/6e506173-afe2-46d8-a1bb-b47e5848ab93)

Unable to check for OTA, same error as above.

Also the physical button on the device doesn't seem to update the state on z2m. And hitting "refresh" on the state doesn't fetch the new state.
UPDATE - After sitting on my network for like an hour, this is no longer an issue. The button does update the state. However I noticed that child lock doesn't lock the button. Will keep an eye on it for phantom solutions again!

I was a bit surprised to see it as a polling only energy monitoring plug, not a massive problem as you can lower the polling interval but is that also possibly a side effect of the model confusion?

Happy to provide any logs for anything needed.
Zigbee2MQTT version 1.32.1

**Comment by Norin-Radd** [comment]
Same issue

and some entities are not available.

Edit, is there some sort of fix for this?

**Comment by sanderjohan** [comment]
I am experiencing the same issue. For me all entities are there and updating. Also the child lock is working.
But OTA is giving an error.

**Comment by dopeytree** [comment]
How did you pair these? 

I bought these 'wooley zigbee ‎BSD29 smart plugs' but the box is for a wifi plug, but the sticker says zigbee. Also no wifi or bluetooth shows up so I think it is zigbee... 

The actual device says model: SA-003 

https://www.amazon.co.uk/dp/B0CDGHVK6N?ref=ppx_yo2ov_dt_b_product_details&th=1
